t/venom/pfservers/configurator/step4.yml

```diff
@@ -0,0 +1,34 @@
+name: Step 4
+version: 2
+testcases:
+- name: start_packetfence
+  steps:
+  - script: '/usr/local/pf/bin/pfcmd service pf start'
+
+- name: start_fingerbank
+  steps:
+  - script: 'systemctl start packetfence-fingerbank-collector'
+
+- name: start_fingerbank
+  steps:
+  - script: 'systemctl status packetfence-fingerbank-collector'
+    assertions:
+      - result.code ShouldEqual 0
+
+- name: disable_configurator
+  steps:
+  - script: "printf '\n[advanced]\nconfigurator=disabled\n' >> /usr/local/pf/conf/pf.conf"
+
+- name: configreload_hard
+  steps:
+  - script: "/usr/local/pf/bin/pfcmd configreload hard"
+
+- name: check_redirection
+  steps:
+  - script: "curl {{.pfserver_webadmin_url}}/api/v1/configurator/config/interfaces -k"
+    assertions:
+      - result.systemout ShouldContainSubstring 'The configurator is turned off'
+
+- name: restart_server
+  steps:
+  - script: 'shutdown -r now'
```

Comment:
Same remark as for step 0, you should use API configurator in place of running commands using script executor.

Reply:
Restart is not mandatory if we set the hostname through Ansible.